HKG Longitudinal Tune

Discountchubbs opened the pull request:
Significantly cleaner injections for HKG longitudinal tune. Transparency over what the code does in each step. Updates will be provided shortly for each addition (Radar tracks and lead distance), followed by updates for the car dashboard. All will focus on clean, transparent code injections with minimal global code changes.

Discountchubbs committed:
Longitudinal_Tune test

FrogAi commented:
Thank you for your PR! If you're not already in the FrogPilot Discord, [feel free to join](https://discord.FrogPilot.download) and let me know you've opened a PR!

Discountchubbs committed:
Brake Light indicator as speed value

Discountchubbs committed:
Update Longitudinal tune

Discountchubbs committed:
#param radar tracks since that logic isnt in this PR

Discountchubbs committed:
Update longitudinal tuning

Updates longitudinal tuning. In hyundaican.py you can enable the cb_upper/lower again for frogpilot_testing. I fixed the code, but want more testers (3 have tested comfort bands so far, and have good reviews). If this is applied directly to staging, then leave cb_upper/lower at 0.0 for now like it is

Discountchubbs commented:
@FrogAi Ready for review. Note the commit description:

 In Hyundaican.py you can enable the cb_upper/lower again for frogpilot_testing. I fixed the code, but want more testers (3 have tested comfort bands so far, and have good reviews). If this is applied directly to staging, leave cb_upper/lower at 0.0 for now like it is.
 
 Comfort bands are mystery code, and you have to zoom in on the cabana to see the signals because it's such a small factor (i.e., between 0 and 1). Therefore, even though a few of us have tested the new CB addition, we should keep it as 0.0 for now in Hyundai CAN and maybe open a test branch or toggle to allow users to try out CB. Why? CB makes cruise control so much smoother at braking that it is out of this world.
